### PR TITLE
New documentation rendering

### PR DESCRIPTION
**Pull request by jmaerki**
As per the last telco, I present my first merge request for documentation changes on the 1.1 branch. This work is still ongoing.

### CODE REVIEW

**Comment by sgrossberndt** [comment]
Could you update to the latest state here please?

File: OJP/OJP_FareSupport.xsd [hunk 1]
```diff
@@ -164,7 +164,7 @@
 					<xs:documentation>Phone number for booking.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="MinimumBookingPeriod." type="xs:duration" minOccurs="0">
+			<xs:element name="MinimumBookingPeriod" type="xs:duration" minOccurs="0">
```
Comment (sgrossberndt) [inline OJP/OJP_FareSupport.xsd:167]: Errors in the XSD files should be fixed in separate commits

File: OJP/OJP_PlaceSupport.xsd [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xs:schema xmlns="http://www.vdv.de/ojp" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:siri="http://www.siri.org.uk/siri" targetNamespace="http://www.vdv.de/ojp" elementFormDefault="qualified" attributeFormDefault="unqualified">
 	<xs:annotation>
-		<xs:documentation>OJP/OJP_Places.xsd - Supporting definitions for place information</xs:documentation>
+		<xs:documentation>OJP/OJP_PlacesSupport.xsd - Supporting definitions for place information</xs:documentation>
```
Comment (sgrossberndt) [inline OJP/OJP_PlaceSupport.xsd:4]: Errors in the XSD files should be fixed in separate commits

File: docs/check-ojp-schemas.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (sgrossberndt) [inline docs/check-ojp-schemas.sh:1]: This script must be added to https://github.com/VDVde/OJP/blob/changes_for_v1.1/.travis.yml#L7, `xsltproc` is already defined as a dependency to be installed here https://github.com/VDVde/OJP/blob/changes_for_v1.1/.travis.yml#L4

**Review by sgrossberndt — CHANGES_REQUESTED** [review]

**Comment by jmaerki** [comment]
I think I'm finally done with pretty much all open issues.

**Comment by sgrossberndt** [comment]
https://github.com/VDVde/OJP/pull/163 should be merged first and this pull request rebase afterwards. Codewise it looks fine, @skinkie mentioned that @schlpbch found some issues with tables, maybe he could add them either here (or create an issue)?

File: OJP/OJP_FareSupport.xsd (hunk 1 above)
Comment (sgrossberndt) [reply OJP/OJP_FareSupport.xsd:167]: See https://github.com/VDVde/OJP/pull/163

**Comment by sgrossberndt** [comment]
Hello @jmaerki, please rebase this pull request so we can merge it.

**Comment by jmaerki** [comment]
I'm not a daily Git user and have never worked with rebase and do not yet know exactly how that works. I currently lack the time to educate myself on it in order not to mess up. So I've done what I could and just merged the latest changes on changes_for_v1.1 to the doc_issues branch. I hope this is enough. In case it is not, I've added write access to @sgrossberndt to my repo in the hopes that this helps. Otherwise, please allow a few more days so I can make time to educate myself on "rebase".

**Comment by sgrossberndt** [comment]
Hi @jmaerki, your last action https://github.com/VDVde/OJP/pull/154/commits/9a1083287991f25ed550b27d7f546ff59c2de3fb was exactly the step needed. Thanks

File: .travis.yml [hunk 4]
```diff
@@ -4,7 +4,7 @@ before_script:
   - sudo apt-get install -qq libxml2-utils xsltproc asciidoctor
```
Comment (sgrossberndt) [inline .travis.yml:4]: As asciidoctor is no longer needed it should not be installed at all

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+
+# OJP Documentation
```
Comment (sgrossberndt) [inline docs/README.md:2]: Maybe use `# Generating OJP Documentation` instead to make it more clear from the header already this is not the documentation itself?

File: docs/ojp-to-asciidoc.xsl [hunk 6]
```diff
@@ -27,25 +27,69 @@ from top to bottom. Note, however, that for the actual operation,
 
 (C) cantamen/Dirk Hillbrecht 2017
```
Comment (sgrossberndt) [inline docs/ojp-to-asciidoc.xsl:28]: As asciidoc is no longer used is this xsl file obsolete and can be removed?

File: docs/schema-collection.xml [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+	<xi:include href="../OJP/OJP_Common.xsd"/>
+	<xi:include href="../OJP/OJP_FacilitySupport.xsd"/>
+	<xi:include href="../OJP/OJP_Fare.xsd"/>
+	<xi:include href="../OJP/OJP_FareSupport.xsd"/>
+	<xi:include href="../OJP/OJP_JourneySupport.xsd"/>
+	<xi:include href="../OJP/OJP_ModesSupport.xsd"/>
+	<xi:include href="../OJP/OJP_Places.xsd"/>
+	<xi:include href="../OJP/OJP_PlaceSupport.xsd"/>
+	<xi:include href="../OJP/OJP_Requests.xsd"/>
+    <xi:include href="../OJP/OJP_RequestSupport.xsd"/>
```
Comment (sgrossberndt) [inline docs/schema-collection.xml:12]: indent

**Review by sgrossberndt — CHANGES_REQUESTED** [review]

**Comment by sgrossberndt** [comment]
Hi @jmaerki , could you please update this pull request?

**Comment by jmaerki** [comment]
Done. Sorry for the delay. Busy times.

**Review by sgrossberndt — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Comment by sgrossberndt** [comment]
Rendering can be seen here: https://jmaerki.github.io/OJP/generated/OJP.html#PointOfInterestRefStructure

**Pull request merged**